tests/test_public.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from utils import TestCase
+
+from nacl.bindings import crypto_box_PUBLICKEYBYTES, crypto_box_SECRETKEYBYTES
+from nacl.public import PrivateKey, PublicKey
+
+
+class TestPublicKey(TestCase):
```

Comment:
I'm okay with the added functions but I'd rather they were top level rather than obtained via subclassing. I know there are other parts of pynacl that use inheritance but in general we'd like to move away from that where possible.